1.1/profile.ttl

```diff
@@ -134,43 +143,47 @@
     prof:hasRole role:specification ;
 .
 
-<http://www.opengis.net/def/geosparql/rules-script>
+<http://www.opengis.net/def/geosparql/rifrules-script>
+    a prof:ResourceDescriptor ;
     dcterms:title "Generator script for GeoSPARQL Rules in RIF"@en ;
     dcterms:description "This is a Python script used to generate the 'GeoSPARQL Rules in RIF' artifact from templates specified in the 'Ontology specification in PDF'. This script is not indended to be executable 'on the web'. Its content is presented as plain text for reference"@en ;
     dcterms:format "text/plain" ;
     prof:hasArtifact "https://github.com/opengeospatial/ogc-geosparql/blob/master/1.1/rifgenerator.py"^^xsd:anyURI ;
     prof:hasRole role:script ;  # this may not be appropriate: we may need a new Role, see https://github.com/w3c/dx-prof/issues/41
-
 .
 
-<http://www.opengis.net/def/geosparql/sf-jsonldcontext>
-    dcterms:title "JSON-LD context for the sf namespace"@en ;
-    dcterms:description "This is a JSON-LD context which can be used to incorporate the Simple Features vocabulary in a JSON-LD file"@en ;
-    dcterms:format "text/json" ;
-    prof:hasArtifact "https://github.com/opengeospatial/ogc-geosparql/blob/master/1.1/contexts/sf-context.json"^^xsd:anyURI ;
-    prof:hasRole role:specification ;
+<http://www.opengis.net/def/geosparql/reqs>
+    a prof:ResourceDescriptor ;
+    dcterms:title "GeoSPARQL 1.1 Requirement and Conformance Class Vocabulary"@en ;
+    dcterms:description "A vocabulary of the requirements and conformance classes in the GeoSPARQL 1.1 specification"@en ;
+    dcterms:conformsTo <https://www.w3.org/TR/skos-reference/> ;
+    dcterms:format "text/turtle" ;
+    prof:hasArtifact "https://github.com/opengeospatial/ogc-geosparql/blob/master/1.1/reqs.ttl"^^xsd:anyURI ;
+    prof:hasRole role:specification ;  # this may not be appropriate: we may need a new Role, see https://github.com/w3c/dx-prof/issues/41
 .
 
 <http://www.opengis.net/def/geosparql/geo-jsonldcontext>
+    a prof:ResourceDescriptor ;
     dcterms:title "JSON-LD context for the geo namespace"@en ;
     dcterms:description "This is a JSON-LD context which can be used to incorporate the GeoSPARQL vocabulary in a JSON-LD file"@en ;
     dcterms:format "text/json" ;
     prof:hasArtifact "https://github.com/opengeospatial/ogc-geosparql/blob/master/1.1/contexts/geo-context.json"^^xsd:anyURI ;
     prof:hasRole role:specification ;
 .
 
+<http://www.opengis.net/def/geosparql/sf-jsonldcontext>
+    a prof:ResourceDescriptor ;
+    dcterms:title "JSON-LD context for the sf namespace"@en ;
```

Comment:
It does not seem right to speak of an sf namespace. In my view, a namespace is defined by its IRI, and people are at liberty to use any prefix as shorthand. So I propose changing this to "JSON-LD context for the Simple Features vocabulary", or something similar.

Reply:
Fixed this, and updated the JSON-LD context title for the ontology too, with https://github.com/opengeospatial/ogc-geosparql/pull/288/commits/e181ec1b18639224a3f8509ac9afd2a0da0c5c3f